tracer_test.go

```diff
@@ -214,13 +214,13 @@ func (s *tracerSuite) TestSetOperationName() {
 func (s *tracerSuite) TestSamplerEffects() {
 	s.tracer.(*Tracer).sampler = NewConstSampler(true)
 	sp := s.tracer.StartSpan("test")
-	flags := sp.(*Span).context.flags
-	s.EqualValues(flagSampled, flags&flagSampled)
+	samplingState := sp.(*Span).context.samplingState
+	s.True(samplingState.isSampled())
```

Comment:
there is already isSampled() on the context, don't need to dereference to samplingState